frontend/packages/knative-plugin/src/components/add/__tests__/eventSource-validation-utils.spec.ts

```diff
@@ -4,26 +4,26 @@ import { getDefaultEventingData } from '../../../utils/__tests__/knative-serving
 import { EventSources } from '../import-types';
 
 describe('Event Source ValidationUtils', () => {
-  describe('CronJobSource : Event Source Validation', () => {
-    it('should validate the form data', async () => {
-      const defaultEventingData = getDefaultEventingData(EventSources.CronJobSource);
-      const mockData = _.omit(_.cloneDeep(defaultEventingData), 'data.cronjobsource.data');
-      await eventSourceValidationSchema
-        .isValid(mockData)
-        .then((valid) => expect(valid).toEqual(true));
-    });
+  it('should validate the form data', async () => {
+    const defaultEventingData = getDefaultEventingData(EventSources.CronJobSource);
+    const mockData = _.omit(_.cloneDeep(defaultEventingData), 'data.cronjobsource.data');
+    await eventSourceValidationSchema
+      .resolve({ value: mockData })
```

Comment:
do we need this `resolve` here

Reply:
the validation schema returns type Lazy which is resolves to a promise which in turn resolves into a schema.
so needed to resolve to get the schema